### ISSUE TITLE
Colored text corresponding to icon.

### ISSUE THREAD

**Issue by ooosssay** [organicmaps/organicmaps#7543]
Making text color correspond with the color of the icon will help make the map more colorful and will help users to identify which text corresponds to which selectable item on the map. 

While scrolling throughout the map, the map usually seems a little empty and monotonous. To help with this, adding color to text will help make the map more colorful and more richer than just grey text.

As there can be many multiple point of interests in populated areas it can sometimes be difficult to know what name goes to which POI. Since the text color is the same with the POI icon color, users will know what name goes to which intractable items.

If this feature ever gets implemented I would suggest adding an outline effect to the text so that the text can remain visible over other colors.

![IMG_5917](https://github.com/organicmaps/organicmaps/assets/129241633/c09e56c3-f69b-4782-863b-b641b86de66a)

**Comment by dvdmrtnz** [comment]
Parks and gardens already use this colored text style. Should this be extended to other POIs? CC @organicmaps/styles 

<img src="https://github.com/organicmaps/organicmaps/assets/47610359/ff5b7c70-8908-4031-8470-7155963170b9" width="320"/>

**Comment by ooosssay** [comment]
Yes sir!

**Comment by RedAuburn** [comment]
hard to say without trying it, but should be easy to add! the biggest concern is legibility.

**Comment by ooosssay** [comment]
That's true, especially because we need to factor in different languages

**Comment by pastk** [comment]
As rightly noted it won't work without an outline. And adding an outline will make text labels stand out noticeably more than they're now.
So we might have to re-balance by adjusting font sizes.

**Comment by ooosssay** [comment]
I realize that this will make some text legibility worse but…
Possibly have poi icons also render with the color it was tagged with. For example in a portion of airports, parking lots are usually color coded and having the color rendered in OM can help users identify which parking lot they are looking for. (This could also help with unnamed but colored parking lots) 
But due to the vast color range that is available text can become unreadable so therefore I emphasize text outline.
